src/app/pages/index.js

```diff
@@ -24,19 +14,51 @@ const applyBasicPageHandlers = pipe(
   withPageWrapper,
   withContexts,
 );
+
 const applyErrorPageHandlers = pipe(withLoading, withPageWrapper, withContexts);
 
 export const ArticlePage = pipe(
   applyBasicPageHandlers,
   withVariant,
-)(_ArticlePage);
-export const ErrorPage = applyErrorPageHandlers(_ErrorPage);
-export const FrontPage = pipe(applyBasicPageHandlers, withVariant)(_FrontPage);
-export const MediaAssetPage = applyBasicPageHandlers(_MediaAssetPage);
-export const MostReadPage = applyBasicPageHandlers(_MostReadPage);
-export const PhotoGalleryPage = applyBasicPageHandlers(_PhotoGalleryPage);
-export const LiveRadioPage = applyBasicPageHandlers(_LiveRadioPage);
-export const OnDemandRadioPage = applyBasicPageHandlers(_OnDemandRadioPage);
-export const OnDemandTvPage = applyBasicPageHandlers(_OnDemandTvPage);
-export const StoryPage = applyBasicPageHandlers(_StoryPage);
-export const IdxPage = applyBasicPageHandlers(_IdxPage);
+)(loadable(() => import(/* webpackPrefetch: true */ './ArticlePage')));
+
+export const ErrorPage = applyErrorPageHandlers(
+  loadable(() => import('./ErrorPage')),
+);
+
+export const FrontPage = pipe(
+  applyBasicPageHandlers,
+  withVariant,
+)(loadable(() => import(/* webpackPrefetch: true */ './FrontPage')));
+
+export const MediaAssetPage = applyBasicPageHandlers(
+  loadable(() => import(/* webpackPrefetch: true */ './MediaAssetPage')),
+);
+
+export const MostReadPage = applyBasicPageHandlers(
+  loadable(() => import(/* webpackPrefetch: true */ './MostReadPage')),
+);
+
+export const PhotoGalleryPage = applyBasicPageHandlers(
+  loadable(() => import('./PhotoGalleryPage')),
+);
+
+export const LiveRadioPage = applyBasicPageHandlers(
+  loadable(() => import('./LiveRadioPage')),
+);
+
+export const OnDemandRadioPage = applyBasicPageHandlers(
+  loadable(() => import('./OnDemandRadioPage')),
+);
+
+export const OnDemandTvPage = applyBasicPageHandlers(
+  loadable(() => import('./OnDemandTvPage')),
+);
+
+export const StoryPage = applyBasicPageHandlers(
+  loadable(() => import(/* webpackPrefetch: true */ './StoryPage')),
+);
+
+export const IdxPage = applyBasicPageHandlers(
+  loadable(() => import(/* webpackPrefetch: true */ './IdxPage')),
```

Comment:
With an IDX page being an index, is it likely someone would go to an IDX page via an onward journey? Perhaps we shouldn't prefetch and IDX?